api_confluence.py

```diff
@@ -168,16 +173,38 @@ def page_payload_write(update_payload):
         print(update_response.text)
 
 
+def pages():
+    """
+    iterates over confluence YAML files and generates pages
+    """
+    for filepath in glob.glob(f"{PATH_YAML_FILES}/*.yaml"):  # Only YAML files
+        with open(filepath, 'r', encoding='utf-8') as file:
+            print(f"LOAD CONFIG FILE - {filepath}")
+            config = yaml.safe_load(file)
```

Comment:
all YAML `config` data gets pre-loaded into variables here.  Those will be passed into various functions to create / update the page content.